src/tests/ftest/util/agent_utils.py

```diff
@@ -148,6 +178,23 @@ def dump_attachinfo(self, output="uri.txt"):
         self.sub_command_class.output.value = output
         return self._get_result()
 
+    def support_collect_log(self, **kwargs):
+        """support collect-log command run.
+
+        Args:
+            kwargs (dic): option to pass to support collect-log command
+
+        Returns:
+            CmdResult: Object that contains exit status, stdout, and other
+                information.
+
+        Raises:
+            CommandFailure: if the daos_agent support collect-log command fails.
+
+        """
+        self.set_command(("support", "collect-log"), **kwargs)
+        return self._get_result()
```

Comment:
Any reason to not use `self._get_json_result` here? Phil has been updating almost all functions to use json by default